_i18n/nb-no/resources/user-guides/multisig-messaging-system.md

```diff
@@ -0,0 +1,864 @@
+
+Slettede tokens kan ikke gjenopprettes eller konstruerer på nytt, i og med at de er tilfeldige. Hvis du er «config-lederen» og sletter tokens, vil du aldri kunne motta auto-config-meldinger igjen som andre signerere sender til deg ved å bruke de slettede tokensene. (Ingen andre vil imidlertid motta dem heller.) Alle vil trenge nye tokens som er utstedt av deg.
+
+### mms send\_signer\_config
+
+    mms send_signer_config
+
+Denne kommandoen sender manuelt din komplette signererkonfigurasjon til alle andre signerere som meldinger av typen `signer config`. Etter å ha mottatt melding din, vil de kunne erstatte signererkonfigurasjonen sin med din med en `mms next`-kommando. Det kommer en sikkerhetstilskyndelse før dette skjer.
+
+Hver signerer og vil få sin merkelappen sin overskrevet med merkelappen du la inn for dem, men sin egen Bitmessage- og Monero-adresse vil bevares.
+
+Denne kommandoen og dens mulighet til å «kringkaste» en bestemt signererkonfigurasjon kan tjene som en byggestein for noe som en «semi-auto-config». Se også kapittelet *Sending av signererkonfigurasjon*. Å sende uten en komplett signererkonfigurasjon er også en del av en helautomatisk konfigurasjon, selv om man ikke trenger en separat `mms send_signer_config`-kommando for det.
+
+## Sikkerhet
+
+MMS-en ble nøye utformet og implementert som et system som tilbyr en høy grad av sikkerhet.
+
+Noe som for øvrig ikke var spesielt lett: Monero multisig er i seg selv allerede en mangesidig, om ikke annet kompleks situasjon, og derfor ikke triviell å sikre, og MMS-en er et kraftfull, for ikke å nevne komplekst system, så det er ikke rart at det finnes forskjellige mulige sikkerhetsproblemer.
+
+Mer at dette er den **første** versjonen av MMS-en, og det kan være slik at folk som bruker den på forskjellige måter vil avdekke nye sikkerhetsproblemer utover de som er nevnt her eller la noen av dem vises i et annet lys. Det er imidlertid grunn til håp om at MMS-en ikke har noen dype og grunnleggende «ureparerbare», konseptuelle mangler.
+
+Kortversjon: Hvis du er i tvil, begynn kun å bruke MMS-en etter du har konfigurert multisig-lommebøkene dine på egenhånd, og formodentlig på sikrere måter enn MMS-en kan tilby (ikke trivielt, men gjennomførbart). Hvis i enda mer tvil, ikke bruk MMS-en.
+
+### Bruk av kryptering og signaturer
+
+All meldingsinnhold krypteres ved å enten bruke Monero-visningsnøklene til signerernes Monero-lommebøker eller med tilfeldig genererte nøkler av samme styrke i tilfelle auto-config-meldingsinnhold. Dette kan virke litt overdrevent med tanke på at PyBitmessage allerede krypterer alle meldingene selv, men PyBitmessage er for det første en tredjepartsprogramvare som du kanskje ikke vil stole på, og for det andre er MMS-en med denne funksjonen til en viss grad allerede forberedt på mindre sikre kommunikasjonssystemer som ikke krypterer seg selv.

```

Comment:
Typo in first word: Alt*. Also, change to "… i tilfelle av auto-config- …"

Reply:
fixed